apps/wallet/src/shared/utils/bip39.test.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { describe, it, expect } from 'vitest';
+
+import { normalizeMnemonics, validateMnemonics } from './bip39';
+
+describe('mnemonics', () => {
+    it('normalize mnemonics', () => {
+        expect(normalizeMnemonics(' Almost a Seed    Phrase')).toEqual(
+            'almost a seed phrase'
+        );
+    });
+
+    it('valid mnemonics', () => {
+        expect(validateMnemonics('result')).toBe(false);
+        expect(validateMnemonics('aaa')).toBe(false);
+        expect(
+            validateMnemonics(
+                'random result crisp session latin must fruit genuine question prevent start coconut brave speak student dismiss'
```

Comment:
I'd personally add tests with 12 word mnemonics, as this is the most common, unless we enforce 16 at minimum (which I don't think we will).

Ideally a test that works for all acceptable sizes of mnemonics would also help.